CHANGELOG.md

```diff
@@ -7,14 +7,10 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 ## Unreleased
 
 ### Breaking Changes
```

Comment:
not sure if these should be set as "breaking"
it changes the structure of the message, but does so in as "non-breaking" a way as possible, adding new fields rather than changing anything about existing fields

Reply:
I think this falls squarely into non-breaking.

Reply:
moved to changed